lib/project/results-model.coffee

```diff
@@ -67,9 +67,6 @@ class ResultsModel
   onDidRemoveResult: (callback) ->
     @emitter.on 'did-remove-result', callback
 
-  onDidFinishSearching: (callback) ->
```

Comment:
It's a duplicate - it's already defined at https://github.com/mnquintana/find-and-replace/blob/coffeelint/lib/project/results-model.coffee#L46-L47